generator.go

```diff
@@ -66,15 +85,9 @@ func (t *Generator) CreateToken(data Data, options *Option) (string, error) {
 		return "", err
 	}
 
-	// generate the encoded headers
-	encodedHeader, err := encodedHeader()
-	if err != nil {
-		return "", err
-	}
-
 	// setup the claims for the token
 	claim := struct {
-		*Option
+		*Option  `json:"Options,omitempty"`
```

Comment:
If we have the `json` tag then it puts the `Option` struct under an `Options` node in the JSON string. It'll fail Firebase's authentication